otter/worker/launch_server_v1.py

```diff
@@ -308,6 +308,13 @@ def _check_server_created(f):
         d.addBoth(_check_results, f)
         return d
 
+    def _create_with_delay():
+        d = _treq.post(path, headers=headers(auth_token),
+                       data=json.dumps({'server': server_config}), log=log)
+        # Add 1 second delay to space 1 second between server creations
+        d.addCallback(delay, clock, 1)
```

Comment:
This is the easiest way I thought of throttling requests on a single node. Another way is to space out job creation in [supervisor](https://github.com/rackerlabs/otter/blob/master/otter/supervisor.py#L559). Is this what you were suggesting? This is probably simpler but doesn't ensure spacing when multiple policies (of possibly different groups) are getting executed at same time. I can understand if that is considered an over-optimization at this point.